### ISSUE TITLE
ttp:contentProfiles missing from IMSC1.1 Text Profile output option

### ISSUE THREAD

**Issue by jbaden** [sandflow/ttconv#369]
Hello, 

The TTCONV documentation references [IMSC1.1 Text Profile](https://www.w3.org/TR/ttml-imsc1.1/#text-profile) as one of the three supported output formats (the others being SRT and WebVTT). 

However, when using `--otype: TTML` there is no declaration written into the `<tt> ` element to signal the appropriate, corresponding profile. Can `ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text` be added to the `<tt>` element when `--otype: TTML` is specified? 

This is causing some processing issues since we are reliant upon the presence of this attribute/value. 

Thanks!
-Justin

**Comment by palemieux** [comment]
There are several distinct issues:
- the presence of `ttp:contentProfiles` would make the document incompatible with EBU-TT-D, which is perhaps undesirable for some applications. This can be addressed by putting the signaling behind a config parameter
- the TTML writer does not currently guarantee that the resulting document is an IMSC 1.1 document, although it is when using an SCC or EBU STL input. It is a larger undertaking to ensure that the output is always IMSC 1.1.

**Comment by jbaden** [comment]
Thank you, @palemieux. Understood regarding the points above. 

Having the ability to specify the profile using a config parameter would be great!

**Comment by cconcolato** [comment]
Maybe the TTML could writer emit a warning or even fail if IMSC1.1 is requested but not possible?

**Comment by palemieux** [comment]
Yes, in IMSC 1.1 mode, the TTML writer should both convert to IMSC 1.1 when possible, e.g. when `c` units are used, and fail when that conversion is not possible.

**Comment by cconcolato** [comment]
It would be great to have a config option that would set the contentProfiles attribute with the IMSC profile when ttconv knows it is generating IMSC-compliant content, for example when the source is STL.

**Comment by palemieux** [comment]
@jbaden see #461
